diesel/src/query_builder/select_statement/dsl_impls.rs

```diff
@@ -86,6 +86,36 @@ where
     }
 }
 
+#[cfg(feature = "postgres")]
```

Comment:
Not sure this (the generic `query_builder` mod) is the right place to a postgres specific impl?

Reply:
So better move this to pg/query_builder/query_fragment_impl.rs?

Reply:
Yeah, but maybe someone else has a good idea for how to organize these impls?